### DISCUSSION TITLE
bg-light not toggling well between themes

### DISCUSSION THREAD

**Discussion by akulmehta** [twbs/bootstrap#37705] [Q&A] [answered]
Super excited with [5.3.0-alpha1](https://blog.getbootstrap.com/2022/12/24/bootstrap-5-3-0-alpha1/) and I went ahead and installed it. I noticed that elements with bg-white and bg-light do not toggle well into dark mode i.e. text becomes white and unreadable etc. 

Would like to know what (if any) is the recommended solution for this?

See examples below

- Nav Bar: notice the brand name

Light mode:

![image](https://user-images.githubusercontent.com/42812604/209448049-e553387c-eec4-4d45-b2f7-11310e039a14.png)

Dark mode:
![image](https://user-images.githubusercontent.com/42812604/209448061-d789b8ed-3ec0-4e3b-828c-879d5c2007dd.png)

- Cards: notice the heading "Vendor"

Light Mode:

![image](https://user-images.githubusercontent.com/42812604/209448090-71f228e9-0f08-4161-a861-b2542a1a39b1.png)

Dark Mode:

![image](https://user-images.githubusercontent.com/42812604/209448083-9f8f4f66-1208-4516-80d6-f5403161c12e.png)

**Comment by mdo** [comment, marked as answer]
This is correct behavior right now, but we could change it. Changing "white" to black or similar doesn't make sense. We recommend using `bg-body` instead of `bg-white` for that one.

For `bg-light`, use `bg-body-secondary` or `bg-body-tertiary`. These new utilities and variables are designed to respect color modes better.
